compiler/rustc_codegen_cranelift/example/mini_core_hello_world.rs

```diff
@@ -55,6 +55,11 @@ struct NoisyDrop {
     inner: NoisyDropInner,
 }
 
+struct NoisyDropUnsized {
+    text: str,
+    inner: NoisyDropInner,
```

Comment:
I believe the text field needs to be last. Note that rustc_codegen_cranelift's tests are currently not run on rust's CI.

Reply:
Yes apparently the field does need to be last. I assumed that `#[repr(Rust)]` allowed arbitrary field ordering in source.